plugins/out_opensearch/opensearch.c

```diff
@@ -1219,6 +1255,13 @@ static struct flb_config_map config_map[] = {
      "When enabled print the OpenSearch exception to stderr (for diag only)"
     },
 
+    /* HTTP Compression */
+    {
+     FLB_CONFIG_MAP_STR, "compress", NULL,
+     0, FLB_FALSE, 0,
```

Comment:
Please add an appropriately named member to the `flb_opensearch` structure and set it here so you can access it in `flb_os_conf_create` instead of using `flb_output_get_property` which is a deprecated pattern.

Reply:
Thanks for pointing me to the correct way of defining the configuration value. It has been changed in [6945bea](https://github.com/fluent/fluent-bit/pull/7381/commits/6945bea410795c0a67fb94b5f6dbabe9b9a71c91). That commit also contains constants, as suggested.